docs/configuration.md

```diff
@@ -196,7 +196,11 @@ The groups property controls how to visually group audits within a category. For
 
 ## Config Extension
 
-The stock Lighthouse configurations can be extended if you only need to make small tweaks, such as adding an audit or skipping an audit, but wish to still run most of what Lighthouse offers. When adding the `extends: 'lighthouse:default'` property to your config, the default passes, audits, groups, and categories will be automatically included, allowing you modify settings or add additional audits to a pass. See [more examples below](#more-examples) to see different types of extension in action.
+The stock Lighthouse configurations can be extended if you only need to make small tweaks, such as adding an audit or skipping an audit, but wish to still run most of what Lighthouse offers. When adding the `extends: 'lighthouse:default'` property to your config, the default passes, audits, groups, and categories will be automatically included, allowing you modify settings or add additional audits to a pass.
+
+Please note that you can only extend from `lighthouse:default` or `lighthouse:full` using the `extends` property. Other internal configs found in the [lighthouse-core/config](https://github.com/GoogleChrome/lighthouse/tree/master/lighthouse-core/config) directory can be used by importing the config object from file reference, or by using the [`--preset`](https://github.com/GoogleChrome/lighthouse#cli-options) CLI flag.
```

Comment:
we have more or less always intended to open `extends` to any other config, but no one has ever asked for it yet, so we've waited :)

Maybe we should open an issue for that and link it here?

Reply:
@brendankenny 

Perhaps what would be useful is creating a `lighthouse-presets` package (or similar) that has a load of useful preset configs that you can import into your own project. `lighthouse-core` could also bring it in interally as a dependency so it's all managed in one place. 

At that point, if a user wants to extend any of those presets, they can import one and do an object merge with their own config rather than relying on the `extends` property. The problem with `extends` is that you have to look up what the available options are, and name it correctly on file name convention.

Let me know what you think of that idea. I'm happy to help with it, but what should we do in the meantime? Do we start with this updated documentation first and then update again later if and when changes are made to how config is done?

Reply:
> Do we start with this updated documentation first and then update again later if and when changes are made to how config is done?

I think this, definitely. Any progress we can make immediately is a good thing.

Maybe open an issue to discuss the other idea? I guess my only hesitation is I haven't seen a ton of interesting variations on configs out there, so I'm not sure if there are enough to justify a standalone collection, but I could be very wrong :)